### DISCUSSION TITLE
Day of the week on the clock

### DISCUSSION THREAD

**Discussion by WinLin97** [valinet/ExplorerPatcher#2768] [Ideas]
Hello, 

I would like to ask you to add a feature to display the day of the week in the tray.

There are ways on the internet to add the day of the week by changing the date format in the regional settings, but this changes the date for me in all programs and it's logs - terrible. 
I just want to display the day of the week in the taskbar. 
I have tried everything, but nothing is satisfactory.

Is it possible to fine-tune or enable this clock system feature with ExplorerPatcher?

This feature was in the first versions of the system, but now it has been disabled and you can no longer see the day of the week in the system bar.

In early versions of Windows 11 this feature looks like this:
![image](https://github.com/valinet/ExplorerPatcher/assets/10725180/7426c350-7add-4607-94ca-08a62bfd2db2)

But now it's gone:
![image](https://github.com/valinet/ExplorerPatcher/assets/10725180/93f50679-f238-4e15-9b36-459e9cebc42d)

Source of photos of early Windows 11 Build: https://www.theverge.com/2021/6/15/22535123/microsoft-windows-11-leak-screenshots-start-menu

**Comment by pyrates999** [comment]
It shows up on Windows 10.  So if the code is still there for windows 11, maybe.

Try switching to the windows 10 taskbar.

**Comment by milestprower92** [comment]
> Hello,
> 
> I would like to ask you to add a feature to display the day of the week in the tray.
> 
> There are ways on the internet to add the day of the week by changing the date format in the regional settings, but this changes the date for me in all programs and it's logs - terrible. I just want to display the day of the week in the taskbar. I have tried everything, but nothing is satisfactory.
> 
> Is it possible to fine-tune or enable this clock system feature with ExplorerPatcher?
> 
> This feature was in the first versions of the system, but now it has been disabled and you can no longer see the day of the week in the system bar.
> 
> In early versions of Windows 11 this feature looks like this: ![image](https://private-user-images.githubusercontent.com/10725180/300923833-7426c350-7add-4607-94ca-08a62bfd2db2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY2NzI0NjIsIm5iZiI6MTcwNjY3MjE2MiwicGF0aCI6Ii8xMDcyNTE4MC8zMDA5MjM4MzMtNzQyNmMzNTAtN2FkZC00NjA3LTk0Y2EtMDhhNjJiZmQyZGIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTMxVDAzMzYwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQyZDAzZjA4Yjg1MTliNWRkOTAxMTgzMWFjZGJlYWZlYTBmYjhlMzVjYzMwNTgwMmE4ZjAzNTVkZWY5MDU5MTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.52R-yQ3oS5xbdU59V-RAY16i1kNZANsGMxS1jFAqHi0)
> 
> But now it's gone: ![image](https://private-user-images.githubusercontent.com/10725180/300923525-93f50679-f238-4e15-9b36-459e9cebc42d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY2NzI0NjIsIm5iZiI6MTcwNjY3MjE2MiwicGF0aCI6Ii8xMDcyNTE4MC8zMDA5MjM1MjUtOTNmNTA2NzktZjIzOC00ZTE1LTliMzYtNDU5ZTljZWJjNDJkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTMxVDAzMzYwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI1NzkxZDE0ZTVkNDg5MWRkYmViM2YwMGFlZDg5ZGNkOGRhMzhhYjQ4MTJiNjcxNGJiYWU5NTY1OWFmNTZmNWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jaZYix_lHjJ8Xik_F8ODjE4Ppwl8rbnejSuNKzR3k-4)
> 
> Source of photos of early Windows 11 Build: https://www.theverge.com/2021/6/15/22535123/microsoft-windows-11-leak-screenshots-start-menu

you can enable that if you increase your taskbar size (Windows 10 Taskbar)

**Reply by WinLin97** [reply]
Yes, thank You all for your help, but I finally found what I needed. 
https://github.com/marticliment/ElevenClock/releases/tag/4.3.3
With this program You can do what you want with your taskbar clock.
![image](https://github.com/valinet/ExplorerPatcher/assets/10725180/871ab8a6-209f-4bd3-be39-b6ab9af3f2cd)
